src/streams/StreamCard.js

```diff
@@ -36,6 +36,7 @@ export default class StreamCard extends PureComponent<Props> {
 
     const name = subscription.name || stream.name;
     const description = subscription.description || stream.description;
+    const subscribed = subscription !== NULL_SUBSCRIPTION;
```

Comment:
I would probably write `isSubscribed` instead of `subscribed`, to be explicit that it's a boolean.

Hmm, also: is there another way to compute it that doesn't use `NULL_SUBSCRIPTION`? It's marked as deprecated, so I imagine there might be a scenario where we remove one or more uses of it and end up with a surprise here (where we think `subscription` might be `===` to it but it isn't).

Reply:
(Same things on `StreamSettingsScreen` in a different commit.)

Reply:
Renamed! Will look into not using `NULL_SUBSCRIPTION`.

Reply:
Removed NULL_SUBSCRIPTION from `StreamSettingsScreen`. Still using `NULL_SUBSCRIPTION.color` in `StreamCard`, but I don't think that's a problem, that'll be easy to remove when we feel like removing `NULL_SUBSCRIPTION` altogether, and in the meantime, it's nice to have it in one place.